app/services/domain/workload-points.js

```diff
@@ -0,0 +1,148 @@
+      .isInt(0, 999)
+    FieldValidator(this.cusB1, 'cusB1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.cusB2, 'cusB2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.cusC1, 'cusC1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.cusC2, 'cusC2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.cusD1, 'cusD1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.cusD2, 'cusD2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commA, 'commA', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commB1, 'commB1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commB2, 'commB2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commC1, 'commC1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commC2, 'commC2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commD1, 'commD1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.commD2, 'commD2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licA, 'licA', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licB1, 'licB1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licB2, 'licB2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licC1, 'licC1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licC2, 'licC2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licD1, 'licD1', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.licD2, 'licD2', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.sdr, 'sdr', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.sdrConversion, 'sdrConversion', errors)
+      .isRequired()
+      .isInt(0, 999)
+    FieldValidator(this.nominalTargetPso, 'nominalTargetPso', errors)
+      .isRequired()
+      .isInt(0, 9999)
+    FieldValidator(this.nominalTargetPo, 'nominalTargetPo', errors)
+      .isRequired()
+      .isInt(0, 9999)
+    FieldValidator(this.weightingOverdue, 'weightingOverdue', errors)
+      .isRequired()
+      .isFloat(0, 100.0)
+    FieldValidator(this.weightingUpw, 'weightingUpw', errors)
+      .isRequired()
+      .isFloat(0, 100.0)
+    FieldValidator(this.weightingWarrants, 'weightingWarrants', errors)
+      .isRequired()
+      .isFloat(0, 100.0)
+    FieldValidator(this.defaultContractedHoursPo, 'defaultContractedHoursPo', errors)
+      .isRequired()
+      .isInt(0, 37)
+    FieldValidator(this.defaultContractedHoursPso, 'defaultContractedHoursPso', errors)
+      .isRequired()
+      .isInt(0, 37)
+    FieldValidator(this.parom, 'parom', errors)
+      .isRequired()
+      .isInt(0, 999)
```

Comment:
Again, are these all agreed boundaries?

Reply:
This is based off the max values for the historical data. All points values are three digits max.

Contracted Hours is a question I've raised. Current known value is 37, but could possibly change in future.